nartica/map.xml

```diff
@@ -93,22 +93,22 @@
     <tool>iron sword</tool>
     <tool>bow</tool>
 </toolrepair>
-<flags flag-proximity-metric="none">
```

Comment:
Leave this here, this is already disabling flag proximity, adding it to each sub-element isn't helpful.
You need to add `net-proximity-metric="none"` here to also disable proximity for returning flags

Reply:
fixed